entities-search/src/test/scala/io/renku/entities/searchgraphs/commands/QuadsEncodersSpec.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.entities.searchgraphs.commands
+
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpec
+
+class QuadsEncodersSpec extends AnyWordSpec with should.Matchers {}
```

Comment:
Is this intentionally empty?

Reply:
This bit PRs... I simply forgot about adding tests :)

Reply:
Wrong, actually, not the tests are in the `EncodersSpec`